doc/sphinx/examples_python/pair_potential_efield_sensitivity.py

```diff
@@ -47,7 +47,7 @@ def getSystems(distance, system_one, fieldshift):
     system_two.diagonalize(1e-3)
 
     # Restrict the calculated eigenenergies
-    system_two.restrictEnergy(fieldshift - 0.015, fieldshift + 0.015)
+    system_two.restrictEnergy(fieldshift.real - 0.015, fieldshift.real + 0.015)
```

Comment:
Why is `.real` required here and in other files? Isn't `SystemOneReal` a specialization for real Hamiltonians?

Reply:
Very good point. I need to investigate.

Reply:
Turns out this was superfluous. I guess I just added a cautionary `.real` after it failed at some intermediate stage when everything turned into `System...Complex` after I removed the `#ifdefs`.